# 停止参赛，回测错误，请教帮助

平时跑策略都很正常，但是选择参赛模式就报错，麻烦各位大神看一下，谢谢！

修改成下面的配置，就报错
start = '2012-07-01'
end = '2015-08-01'
benchmark = 'HS300'
capital_base = 1000000
csvs = []
security_base = {}
commission = Commission(buycost=0.0008, sellcost=0.0018)  # 佣金万八
slippage = Slippage() 
universe = set_universe('A')
freq = 'd'                        # 策略类型，'d'表示日间策略使用日线回测，'m'表示日内策略使用分钟线回测
refresh_rate = 10                  # 调仓频率，表示执行handle_data的时间间隔，若freq = 'd'时间间隔的单位为交易日，若freq = 'm'时间间隔为分钟

Exception                                 Traceback (most recent call last)
&lt;mercury-input-3-b81414f07ee0&gt; in &lt;module&gt;()
    205                                                           slippage = slippage,
    206                                                           refresh_rate = refresh_rate,
--&gt; 207                                                           freq = freq)
    208         perf = quartz.perf_parse(bt, quartz_acct)
    209 

/home/ipython/anaconda/lib/python2.7/site-packages/quartz/backtest_new.pyc in quick_backtest_generator_new(sim_params, strategy, idxmap_all, data_all, commission, slippage, refresh_rate, freq, *args, **kwargs)
    532         try:
    533             if len(data) and account.days_counter % refresh_rate == 0:
--&gt; 534                 account.handle_data(data)
    535             report.update(account, data)
    536 

/home/ipython/anaconda/lib/python2.7/site-packages/quartz/sim_condition/env.pyc in handle_data(self, data, trigger)
    409         self.referencePortfolioValue = self._get_reference_portfolio_value()
    410         if trigger:
--&gt; 411             self.strg._handle_data(self)
    412 
    413         if hasattr(self, '_current_minute'):

&lt;mercury-input-3-b81414f07ee0&gt; in handle_data(account)
     54     # print(len(filterList))
     55 
---&gt; 56     temp = DataAPI.FdmtEfGet(secID=filterList, forecastType='22', publishDateBegin=yesterday, publishDateEnd=yesterday, field=['secID', 'publishDate', 'NIncAPChgrLL', 'NIncAPChgrUPL'], pandas="1")
     57     temp['meanGrowRate'] = (temp['NIncAPChgrLL'] + temp['NIncAPChgrUPL']) / 2
     58     temp.sort('meanGrowRate', ascending=False).drop_duplicates('secID' ,inplace = True)

/home/ipython/anaconda/lib/python2.7/site-packages/DataAPI/DATAYES.pyc in FdmtEfGet(ticker, secID, reportType, endDate, beginDate, publishDateEnd, publishDateBegin, forecastType, field, pandas)
  12674         csvString = api_base.__getCSV__(''.join(requestString), httpClient, gw=False)
  12675         if csvString is None or len(csvString) == 0 or (csvString[0] == '-' and csvString[1] != '1') or csvString[0] == '{':
&gt; 12676             api_base.handle_error(csvString, 167)
  12677         elif csvString[:2] == '-1':
  12678             csvString = ''

/home/ipython/anaconda/lib/python2.7/site-packages/DataAPI/api_base.pyc in handle_error(csvString, api_id)
     85         result = '\nPlease try again, or check the input parameters'
     86     err_msg = ('%s for request %s' % (csvString if csvString is not None else 'Query Error', result))
---&gt; 87     raise Exception(err_msg)
     88 
     89 def get_token():

Exception: -2:Illegal Request Parameter for request 
Please try again, or check the input parameters